cloudca/resource_cloudca_instance.go

```diff
@@ -276,7 +276,6 @@ func resourceCloudcaInstanceUpdate(d *schema.ResourceData, meta interface{}) err
 		if err != nil {
 			return err
 		}
-		d.SetPartial("compute_offering")
```

Comment:
dont know if its ok that i just deleted these - didnt see any notes about how to replace them when they were deprecated